Packs/RetarusSecureEmailGateway/Integrations/RetarusSecureEmailGateway/RetarusSecureEmailGateway_description.md

```diff
@@ -0,0 +1,3 @@
+## BaseIntegration Help
```

Comment:
Please correct this.
Notice that this will show as help section, meaning where to get the params from and how to configure the instance.